src/d/actor/d_a_kui.cpp

```diff
@@ -4,68 +4,541 @@
 //
 
 #include "d/actor/d_a_kui.h"
+#include "d/actor/d_a_player.h"
+#include "d/actor/d_a_btd.h"
+#include "d/actor/d_a_dr2.h"
+#include "d/d_bg_s_movebg_actor.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_kankyo_rain.h"
 #include "d/d_procname.h"
+#include "d/d_s_play.h"
+#include "d/res/res_kui.h"
+#include "f_op/f_op_camera.h"
+#include "f_op/f_op_actor_mng.h"
+#include "f_pc/f_pc_executor.h"
+#include "JSystem/JUtility/JUTReport.h"
 
 class J3DModelData;
 
 /* 00000078-000000C4       .text s_a_i_sub__FPvPv */
-void s_a_i_sub(void*, void*) {
-    /* Nonmatching */
+static void* s_a_i_sub(void* i_this, void*) {
```

Comment:
The first param to search functions is the actor it's checking against to see if it matches, not the `this` actor. So let's name it something else like:
```suggestion
static void* s_a_i_sub(void* search, void*) {
```